src/components/BoxCard.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import githublogo from "../svgs/github.svg";
+import gradientCircle from "../svgs/gradientCircle.svg";
+import githubStar from "../svgs/githubStar.svg";
+
+const BoxCard = ({
+  contentType,
+  title,
+  subtitle,
+  description,
+  link,
+  urlLink,
+  step,
+  githubRepo,
+  githubOrg,
+}) => {
+  const cols = contentType == "project" ? 6 : 4;
+  const [stars, setStars] = useState(0);
+
+  useEffect(() => {
+    if (githubRepo && githubOrg) {
+      fetch(`https://api.github.com/repos/${githubOrg}/${githubRepo}`, {
+        method: "GET",
+      })
+        .then(results => results.json())
+        .then(data => {
+          setStars(data.stargazers_count);
+        });
+    }
+  }, [githubRepo]);
+
+  return (
+    <Card style={{ gridColumn: `auto/span ${cols}` }}>
+      {step && (
+        <>
+          <StepCircle></StepCircle>
```

Comment:
Can we make empty components self closing to keep consistency throughout the file? See line 46.

```suggestion
          <StepCircle />
```

Reply:
Okay